build/Build.cs

```diff
@@ -397,8 +397,8 @@ void CompressCalamariProject(Project project)
                           actions.Add(() =>
                           {
                               var publishedLocation =
-                                  DoPublish("Calamari.Tests", Frameworks.NetCoreApp31, nugetVersion, rid);
-                              var zipName = $"Calamari.Tests.netcoreapp.{rid}.{nugetVersion}.zip";
+                                  DoPublish("Calamari.Tests", Frameworks.Net60, nugetVersion, rid);
+                              var zipName = $"Calamari.Tests.{rid}.{nugetVersion}.zip";
```

Comment:
I've removed the netcoreapp slug here and updated the artifact dependencies from `Calamari.Tests.netcoreapp.%rid%.nuget` -> `Calamari.Tests*.%rid%.nuget` for now to work in the interim for both versions. The * can be removed if this change merges through.